httpie/encoding.py

```diff
@@ -0,0 +1,50 @@
+from typing import Union
+
+from charset_normalizer import from_bytes
+from charset_normalizer.constant import TOO_SMALL_SEQUENCE
+
+UTF8 = 'utf-8'
+
+ContentBytes = Union[bytearray, bytes]
+
+
+def detect_encoding(content: ContentBytes) -> str:
+    """
+    We default to utf8 if text too short, because the detection
+    can return a random encoding leading to confusing results:
+
+    >>> too_short = ']"foo"'
+    >>> detected = from_bytes(too_short.encode()).best().encoding
+    >>> detected
+    'utf_16_be'
```

Comment:
It's not the case anymore using the latest version.

Reply:
Good to know! What has changed?

Reply:
The detection has been improved over tiny content and *obviously* ASCII and Unicode contents. https://github.com/Ousret/charset_normalizer/pull/63 https://github.com/Ousret/charset_normalizer/pull/93 

Also, the library no longer raises any warnings regarding tiny contents. (https://github.com/aio-libs/aiohttp/pull/5930#discussion_r706508076) https://github.com/Ousret/charset_normalizer/pull/92

Reply:
That’s exactly what we needed. Is there still some length threshold below which it’s unreasonable to use rely on the detected encoding? Or a way to get some sort of confidence interval for the best match?

Reply:
Maybe could we require `charset_normalize>=2.0.5` and drop our own `TOO_SMALL_SEQUENCE` check?

Reply:
I committed https://github.com/httpie/httpie/pull/1168/commits/d52a4833e461e1b16b7961a112ea5c53e93cd643 just to see how it goes.

Reply:
Although I cannot guarantee it, the detection regarding tiny ASCII/Unicode content should be very much reliable.
The other cases scenarii (outside ASCII and Unicode (8, 16 & 32) should not be trusted below the `TOO_SMALL_SEQUENCE` threshold length. But I don't see how to provide the end-user with the best readable output besides this.
Do not hesitate to report any edge cases.

Reply:
I'll revert https://github.com/httpie/httpie/commit/d52a4833e461e1b16b7961a112ea5c53e93cd643 as it seems too new.